src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -389,16 +399,9 @@ time (useful for testing the effect of a long-running trigger function
 - which should be avoided) and has a configurable random chance of
 success. The function signature is:
 
-.. code-block:: python
-
-   xrandom(percent, secs=0, _=None, debug=False)
-
-The ``percent`` argument sets the odds of success in any given call;
-``secs`` is the number of seconds to sleep before returning; and the
-``_`` argument (underscore is a conventional name for a variable
-that is not used, in Python) is provided to allow specialization of the
-trigger to (for example) task name, task ID, or cycle point (just use
-the appropriate string templates in the workflow configuration for this).
+.. automodule:: cylc.flow.xtriggers.xrandom
+   :members: xrandom, validate
+   :member-order: bysource
```

Comment:
```suggestion
   :members: xrandom
```

Reply:
True but it does provide a link to the source so it is not useless IMO

Reply:
https://github.com/cylc/cylc-flow/pull/6084